lab_5_scrapper/scrapper.py

```diff
@@ -90,25 +178,37 @@ def __init__(self, config: Config) -> None:
         """
         Initializes an instance of the Crawler class
         """
-        pass
+        self.config = config
```

Comment:
do you think that this attribute should be public?

Reply:
done